Gems/Atom/RPI/Code/Source/RPI.Reflect/Pass/PassAttachmentReflect.cpp

```diff
@@ -67,55 +67,10 @@ namespace AZ
                     ->Field("BufferViewDesc", &PassSlot::m_bufferViewDesc)
                     ->Field("LoadStoreAction", &PassSlot::m_loadStoreAction)
                     ->Field("FormatFallbacks", &PassSlot::m_formatFallbacks)
-                    ->Field("FormatFilter", &PassSlot::m_formatFilter)
-                    ->Field("DimensionFilter", &PassSlot::m_dimensionFilter)
                     ;
             }
         }
 
-        template<typename FilterType>
```

Comment:
I assume this was not used anywhere?

Reply:
Correct, was never used anywhere. Use case was on your pass template you could specify the input should only accept certain formats, for example you could specify a list of color formats to make it clear you expect a color texture as input or a list of depth formats to express you only want a depth texture bound in this slot, then if you try to attach an attachment in the slot that doesn't match the filters, the pass system will warn you about it. It's not something that we've ever used though and the user will get warnings from the RHI and will see that their pass doesn't render correctly, so it's just extra work to get a warning you'd get anyway.